e2e/dotcms-e2e-node/frontend/tests/contentSearch/contentEditing.spec.ts

```diff
@@ -117,4 +117,92 @@ test('Validate you are able to add file assets importing from url', async ({page
     //fileName?: string, fromURL?: string, newFileName?: string, newFileText?: string) {
     await contentUtils.workflowExecutionValidationAndClose(page, 'Content saved');
     await contentUtils.validateContentExist(page, fileAssetContent.title).then(assert);
-});
\ No newline at end of file
+});
+
+/**
+ * Test to validate you are able to add file assets creating a new file
+ */
+test('Validate you are able to add file assets creating a new file', async ({page}) => {
+    const contentUtils = new ContentUtils(page);
+    const iframe = page.frameLocator(iFramesLocators.main_iframe);
+
+    await contentUtils.addNewContentAction(page, fileAsset.locator, fileAsset.label);
+    await contentUtils.fillFileAssetForm(page, fileAssetContent.host, fileAssetContent.title, contentProperties.publishWfAction, null, null, fileAssetContent.newFileName, fileAssetContent.newFileText);
```

Comment:
Consider refactoring `fillFileAssetForm` to accept a single object as its argument when passing more than 2 parameters. 

This will improve readability, especially with the `null` values for unused parameters, which can instead be omitted as optional properties. It also makes the function easier to extend in the future

Reply:
Done, fixed in the new PR...